crates/anvil/core/src/types.rs

```diff
@@ -217,6 +219,7 @@ pub struct AnvilMetadata {
     pub latest_block_number: u64,
     pub latest_block_hash: H256,
     pub forked_network: Option<ForkedNetwork>,
+    pub snapshots: HashMap<U256, (u64, H256)>,
```

Comment:
this should be BtreeMap

Reply:
Updated. but curious, why is this important? or is it just for visual consistency?

(a bit more nitpicky, but perhaps the whole `Backend::active_snapshots` be a btree from the start, to avoid conversion here, and perhaps to slightly optimize its cleanup?

Reply:
> or is it just for visual consistency?

just for that yes

> but perhaps the whole Backend::active_snapshots

yeah that'd be easier